circuits/circom/ecdh.circom

```diff
@@ -1,6 +1,7 @@
 pragma circom 2.0.0;
-include "../node_modules/circomlib/circuits/bitify.circom";
-include "../node_modules/circomlib/circuits/escalarmulany.circom";
+
+include "./bitify.circom";
```

Comment:
How are we able to import this template like this now (vs. referencing circomlib from node_modules)?

Makes it seem like this template is within the current directory 🤔

Reply:
https://github.com/privacy-scaling-explorations/maci/pull/1010#discussion_r1451600519 basically circom allows to map imports, and circomkit supports it directly via its json config file

Reply:
I see - so should we update this across all imports so we do it consistently?

Reply:
Maybe not a big deal but I think a downside of this approach is now it's less obvious where we're importing templates from (i.e. within our codebase vs. libraries)

Reply:
I see what you mean. Dunno really, I quite like it as it's shorter but I definitely see your point. Can always put it back if the team feels like we should keep as is

Reply:
I added comments throughout the templates so it is clear which templates are coming from circomlib/zk-kit vs internal